src/mediumComplexityProblem7/SortArray.java

```diff
@@ -0,0 +1,30 @@
+package mediumComplexityProblem7;
+
+import java.util.Arrays;
+
+public class SortArray {
+    public void ascendingSort(int[] inputArray) {
+        int firstCounter;
+        int secondCounter = 0;
+        Arrays.sort(inputArray);
```

Comment:
Please sort without using collection API.

Reply:
Done!! Added code to sort the array elements.